settings.gradle.kts

```diff
@@ -9,4 +9,11 @@ gradleEnterprise {
 		termsOfServiceUrl = "https://gradle.com/terms-of-service"
 		termsOfServiceAgree = "yes"
 	}
-}
\ No newline at end of file
+}
+
+// See https://community.sonarsource.com/t/sonar-gradle-skipcompile-is-not-working/102710/9
```

Comment:
You can remove this line as SONARGRADL-134 already links to it.

Reply:
will do

Reply:
It was originally meant for keeping even after the TODO is removed, but I realize that if we can move this to a better place, the link will become invalid too, so removed.